app/views/catalog/_other_versions_show_more.html.erb

```diff
@@ -0,0 +1,12 @@
+<% additional_linked_records = linked_record_count - Orangelight.config['show_page']['linked_documents']['maximum'] %>
+
+<% if linked_record_count > Orangelight.config['show_page']['linked_documents']['maximum'] %>
+  <button class="show-more-linked-records btn btn-sm btn-link"
```

Comment:
We should include an aria-expanded to indicate if the button is collapsed or expanded and then update the value accordingly with javascript. I was thinking to add `aria-expanded="false"` since it is collapsed when we load the page; then in JS I would add an event listener and make sure it updates the `aria-expanded` every time you press `enter` or `space bar`. I did something similar for the facets link https://github.com/pulibrary/orangelight/blob/main/app/javascript/orangelight/accessible_facets.js. 
We can include a class 'show-more-related' with an icon for example https://fontawesome.com/v4/icon/angle-double-down which is unicode Unicode: f103 to indicate that it will expand when you click it and a class show-less-related with the https://fontawesome.com/v4/icon/angle-double-up f102 icon to indicate that the user can collapse it. These classes could be updated in the Js you have [displayFewer](https://github.com/pulibrary/orangelight/pull/3101/files#diff-0497c1f630b91369817e989dde756b9b414b8255f7aa63c0a1581ebfd3db5c51R32) and [displayMore](https://github.com/pulibrary/orangelight/pull/3101/files#diff-0497c1f630b91369817e989dde756b9b414b8255f7aa63c0a1581ebfd3db5c51R16).

Reply:
This is the stylesheet we use for the record page and an example https://github.com/pulibrary/orangelight/blob/ac9cee8b7ad9c2b50902323c560261cdc93a3569/app/assets/stylesheets/components/record.scss#L183 to the content we use for the thumbnail icon content in the record page.